<ui version="4.0" >
 <class>antiSpamSettingsClass</class>
 <widget class="QWidget" name="antiSpamSettingsClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>antiSpamSettings</string>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>9</number>
   </property>
   <property name="topMargin" >
    <number>9</number>
   </property>
   <property name="rightMargin" >
    <number>9</number>
   </property>
   <item row="0" column="0" >
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" >
      <property name="leftMargin" >
       <number>5</number>
      </property>
      <property name="topMargin" >
       <number>1</number>
      </property>
      <property name="rightMargin" >
       <number>5</number>
      </property>
      <property name="bottomMargin" >
       <number>1</number>
      </property>
      <property name="horizontalSpacing" >
       <number>0</number>
      </property>
      <property name="verticalSpacing" >
       <number>1</number>
      </property>
      <item row="0" column="0" colspan="2" >
       <widget class="QCheckBox" name="acceptListBox" >
        <property name="text" >
         <string>Accept messages only from contact list </string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QCheckBox" name="notifyBox" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="text" >
         <string>Notify when blocking message</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2" >
       <widget class="QCheckBox" name="authBox" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="text" >
         <string>Do not accept NIL messages concerning authorization</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QCheckBox" name="urlBox" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="text" >
         <string>Do not accept NIL messages with URLs</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2" >
       <widget class="QGroupBox" name="botBox" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="title" >
         <string>Enable anti-spam bot</string>
        </property>
        <property name="checkable" >
         <bool>true</bool>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" >
         <property name="leftMargin" >
          <number>5</number>
         </property>
         <property name="topMargin" >
          <number>1</number>
         </property>
         <property name="rightMargin" >
          <number>5</number>
         </property>
         <property name="bottomMargin" >
          <number>1</number>
         </property>
         <property name="horizontalSpacing" >
          <number>0</number>
         </property>
         <property name="verticalSpacing" >
          <number>0</number>
         </property>
         <item row="0" column="0" >
          <widget class="QLabel" name="questionLabel" >
           <property name="text" >
            <string>Anti-spam question:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QTextEdit" name="questionEdit" >
           <property name="maximumSize" >
            <size>
             <width>16777215</width>
             <height>36</height>
            </size>
           </property>
           <property name="verticalScrollBarPolicy" >
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="0" >
          <widget class="QLabel" name="answerlabel" >
           <property name="text" >
            <string>Answer to question:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" >
          <widget class="QLineEdit" name="answerEdit" >
           <property name="maxLength" >
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item row="4" column="0" >
          <widget class="QLabel" name="rightLabel" >
           <property name="text" >
            <string>Message after right answer:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0" >
          <widget class="QLineEdit" name="rightAnswerEdit" >
           <property name="maxLength" >
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item row="6" column="0" >
          <widget class="QCheckBox" name="replyBox" >
           <property name="text" >
            <string>Don't send question/reply if my status is "invisible" </string>
           </property>
          </widget>
         </item>
         <item row="7" column="0" >
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections>
  <connection>
   <sender>acceptListBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>authBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>255</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel" >
     <x>309</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>acceptListBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>urlBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>373</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel" >
     <x>244</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>acceptListBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>botBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>240</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel" >
     <x>151</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
